content/slides/en.yaml

```diff
@@ -78,57 +111,121 @@ levels:
             code: |-
                 {print} Welcome to your own rock scissors paper!
                 {ask} What will you choose?
+            notes: |-
+                Show that the computer now asks you a question. Fill in an answer and also show that once you fill in the answer, nothing happens.
+                Make your students aware that the computer will only follow your instructions, it will not do anything by itself. This means that if you want the computer to print the answer, you have to give that instruction.
```

Comment:
I think these kind of instruction can be really helpful to grasp programming in Hedy when they don't have much experience yet.